rockhound/__init__.py

```diff
@@ -3,10 +3,22 @@
 from . import version
 from .registry import data_location
 from .etopo1 import fetch_etopo1
-from .prem import fetch_prem
+from .earth_models import fetch_prem
+from .earth_models import fetch_ak135f
+from .earth_models import fetch_iasp91
+from .earth_models import fetch_mean
+from .earth_models import fetch_pema
+from .earth_models import fetch_pemc
+from .earth_models import fetch_pemo
+from .earth_models import fetch_mc35
+from .earth_models import fetch_stw105
+from .earth_models import fetch_tna_sna
```

Comment:
These can actually go in a single line: `from .earth_models import fetch_prem, fetch_ak135f, ...`